tests/tests.cpp

```diff
@@ -1570,10 +1606,18 @@ void testRemove() {
         storage.sync_schema();
         storage.remove_all<Object>();
         
-        auto id1 = storage.insert(Object{ 0, "Skillet"});
+        Object object{0, "Skillet"};
+
+        auto id1 = storage.insert(object);
         assert(storage.count<Object>() == 1);
         storage.remove<Object>(id1);
         assert(storage.count<Object>() == 0);
+
+        id1 = storage.insert(object);
+        assert(storage.count<Object>() == 1);
+        auto object_from_database = storage.get<Object>(id1);
```

Comment:
Please add unit tests for composite/primary key specified by getters and setters

Reply:
@fnc12 wait for reviewing #298 because we got some errors

Reply:
@fnc12 Add unit test for this :)